RedfishEventListener_v1.py

```diff
@@ -46,6 +46,21 @@
     'registries': None
 }
 
+from http.server import BaseHTTPRequestHandler
+from io import BytesIO
+
+
+class HTTPRequest(BaseHTTPRequestHandler):
+    def __init__(self, request_text):
+        self.rfile = BytesIO(request_text)
```

Comment:
I'm getting an exception here when an event is received. I'm sending a test payload like this on my end:

`curl -k http://127.0.0.1 -X POST -d '{"Test": "Test"}' -H "Content-Type: application/json"`

Reply:
Thanks for testing and the feedback.

I had some issue with my test setup.

I have got it now. 

I shall test the same and get back to you. 👍